afs/media/js/views/components/reports/project.js

```diff
@@ -38,16 +47,81 @@ define([
                 self.cards = self.createCardDictionary(cards)
 
                 self.nameCards = {
-                    name: self.cards.name,
-                    identifier: self.cards.Identifier,
-                    exactMatch: self.cards.ExactMatch,
-                    type: self.cards.Classification
-                }
-
+                    name: self.cards?.['name of project'],
+                    identifier: self.cards?.['identifier for project'],
+                    type: self.cards?.['scale of project']
+                };
                 self.descriptionCards = {
-                    statement: self.cards.Statement,
+                    statement: self.cards?.['statement about project'],
+                };
+                self.documentationCards = {
+                    digitalReference: self.cards?.['digital reference to project'],
+                    subjectOf: self.cards?.['source reference work for project'],
                 };
-            }
+                self.substanceCards = {
+                    timespan: self.cards?.['dates of project'],
+                };
+            };
+
+            self.parthoodData = ko.observable({
+                sections: 
+                    [
+                        {
+                            title: 'Parthood', 
+                            data: [{
+                                key: 'parent project', 
+                                value: self.getRawNodeValue(self.resource(), 'part of'), 
+                                card: self.cards?.['parent project'],
+                                type: 'resource'
+                            }]
+                        }
+                    ]
+            });
+            self.temporalData = ko.observable({
+                sections: [
+                    {
+                        title: "Temporal Relations of Project", 
```

Comment:
nit: I've been removing the double quotes from strings - can you remove those here and move to single quotes for strings in this file?

Reply:
fixed